### PR TITLE
update frontmatter to use bibinfo and fix other deprecations

### LINKED ISSUE #508: Update to use `<bibinfo/>`
Fixes deprecation at e.g. https://github.com/TeamBasedInquiryLearning/library/actions/runs/12398010510/job/34609860365#step:5:391

### LINKED ISSUE #510: Find/fix PTX:DEPRECATE for misplaced tasks
https://github.com/TeamBasedInquiryLearning/library/actions/runs/12401969891/job/34622481202?pr=509#step:6:5

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #508

Closes #510

### CODE REVIEW

**Comment by StevenClontz** [comment]
Still a few `PTX:DEPRECATE`s related to misplaced tasks, but I'll defer that needle/haystack to a follow-up PR.

File: source/calculus/source/meta/frontmatter.ptx [hunk 1]
```diff
@@ -67,12 +67,14 @@
...
+    <xi:include href="../../../common/copyright.ptx" />
+  </bibinfo>
+  <titlepage><titlepage-items/></titlepage>
+
 
 
   <colophon xml:id="colophon">
-    <xi:include href="../../../common/website.ptx"/>
-    <xi:include href="../../../common/copyright.ptx" />
     <p>
```
Comment (siwelwerd) [inline source/calculus/source/meta/frontmatter.ptx:78]: I'm not seeing where this is showing up in the Preview anymore

Comment (siwelwerd) [reply source/calculus/source/meta/frontmatter.ptx:78]: Ugh, this was supposed to be attached to the subsequent lines where we acknowledge inclusion of other CC licensed stuff, and NSF support

Comment (siwelwerd) [reply source/calculus/source/meta/frontmatter.ptx:78]: The NSF support is included on the TBIL Resource Library page now, so I think nsf.ptx can actually be deprecated.  But we do need to acknowledge the CC licensed stuff we use

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**